Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -92,23 +88,6 @@ public void setUpStep() throws Exception {
 
         addTextReplacementRule(GLOBAL_ENDPOINT, MOCK_URI + "/");
 
-        int retries = 10;
-        boolean created = false;
-        while (retries > 0) {
-            retries--;
-            try {
-                wireMock = new WireMock(MOCK_HOST, wireMockRule.port());
```

Comment:
Yes, @rule will generate a new one for each test case. Also I think I should delete the wireMock=null .No need for that.